Library/Formula/ncdc.rb

```diff
@@ -14,7 +12,24 @@ class Ncdc < Formula
   depends_on 'pkg-config' => :build
   depends_on 'geoip' => :optional
 
+  stable do
+    url 'http://dev.yorhel.nl/download/ncdc-1.19.tar.gz'
+    sha1 '7f478b7daf09202586b40899cc6beabeb0d23178'
+  end
```

Comment:
We don't need a `stable do` block unless there's e.g. stable-specific dependencies or patches.